src/CopyableInput/CopyableInput.tsx

```diff
@@ -12,6 +12,7 @@ import "../less/forms.less";
 export interface Props extends TextInputProps {
   className?: string;
   enableCopy?: boolean;
+  onClickCopy?: Function;
```

Comment:
Ah sorry, I should have mentioned this originally - it's better to type it explicitly as `() => void` instead of `Function`. When it's typed as `Function` we know it's a function, but nothing about what arguments it takes or what it returns.

Reply:
good call, that seems more reasonable to me as well
6acc2d7